x/payment/types/builder_options.go

```diff
@@ -65,7 +65,7 @@ func InheritTxConfig(builder sdkclient.TxBuilder, tx authsigning.Tx) sdkclient.T
 		builder.SetGasLimit(gas)
 	}
 
-	if feeAmmount := tx.GetFee(); !feeAmmount.AmountOf("utia").Equal(sdk.NewInt(0)) {
+	if feeAmmount := tx.GetFee(); !feeAmmount.AmountOf("ucls").Equal(sdk.NewInt(0)) {
```

Comment:
[note for reviewers] we can't replace this with `app.BondDenom` because it will introduce an import cycle.

We may consider extracting [these constants](https://github.com/rootulp/celestia-app/blob/b92d17d709c688f8f9174f7b868829b70154ed73/app/app.go#L94-L99) to the `appconsts` package to mitigate